source-stripe-native/source_stripe_native/models.py

```diff
@@ -74,20 +79,24 @@ class BaseStripeObject(BaseDocument, extra="allow"):
     """
     Shared properties between all Stripe objects.
     """
+
     NAME: ClassVar[str]
     SEARCH_NAME: ClassVar[str]
 
     id: str
     object: str
+    account_id: str | None = None
```

Comment:
This schema generated from this ends up being a little weird, where it has a default of `null`. I'm not exactly sure the best way to deal with this, but what we have done elsewhere is a this bit: `json_schema_extra=lambda x: x.pop("default")` to not schematize the default value. The end goal is to allow the field to be absent. A schema with a default `null` value will mean that the field is always present, but a `null` is substituted if it is absent. 

To be very precise, I think we want the schema for this to be `type: string`, and have it not show up in the list of required fields. Type `[string, null]` isn't really correct since if it is present, it won't ever be `null`, and will always be a string.